native/src/components/SimpleImage.tsx

```diff
@@ -48,6 +49,10 @@ const SimpleImage = ({
     return <Image source={source} resizeMode={resizeMode} style={style} />
   }
 
+  if (source.endsWith('.svg')) {
```

Comment:
`source.endsWith('.svg')` could be moved to a boolean var, so we only need this once.